web/packages/test/scripts/run-smoketests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eu
+
+source scripts/set-env.sh
+
+pushd $root_dir/smoketest
+
+./make-bindings.sh
+echo "run register token test"
+cargo test --test register_token --release -- --nocapture
```

Comment:
I would build all tests without running them first to catch any build errors early.
```suggestion
echo "build tests"
cargo test --no-run
echo "run register token test"
cargo test --test register_token --release -- --nocapture
```